index.js

```diff
@@ -18,6 +19,28 @@ const TimerMixin = require('react-timer-mixin');
 const DefaultTabBar = require('./DefaultTabBar');
 const ScrollableTabBar = require('./ScrollableTabBar');
 
+class StaticContainer extends Component{
```

Comment:
@cancan101 @skv-headless i made changes and used the external component mentioned